### PR TITLE
Use HTML for docs

### PR DESCRIPTION
**Pull request by yschimke**
Seems to fix the generation, already used by https://square.github.io/workflow/

### CODE REVIEW

**Comment by yschimke** [comment]
Example artifacts https://github.com/square/okhttp/suites/9789234155/artifacts/470790462

**Comment by yschimke** [comment]
@swankjesse request post review - be nice to start testing this when we have a release.

File: deploy_website.sh [hunk 1]
```diff
@@ -24,39 +24,9 @@ git clone $REPO $DIR
...
-  :okhttp-tls:dokkaGfm \
-  :okhttp-urlconnection:dokkaGfm \
-  :okhttp:dokkaGfm
+./gradlew dokkaHtmlMultiModule
 
-# mkdocs doesn't correctly handle the .md paths in package-list, so post-process.
-# See https://github.com/square/okhttp/issues/7338
-find docs/4.x -name package-list -exec sed -i '' -e 's/index.md$/index.html/' -e 's/\.md$/\.html/' -e '1 s/gfm/html/' -e '2 s/md/html/' {} \;
-
-# Dokka filenames like `-http-url/index.md` don't work well with MkDocs <title> tags.
```
Comment (swankjesse) [inline deploy_website.sh:41]: Hooray nuking this trash!

**Review by swankjesse — COMMENTED** [review]
LGTM

**Pull request merged**